packages/compiler-core/src/utils.ts

```diff
@@ -62,7 +62,7 @@ export function isCoreComponent(tag: string): symbol | void {
   }
 }
 
-const nonIdentifierRE = /^\d|[^\$\w]/
+const nonIdentifierRE = /^\d|[^\$\w\u4e00-\u9fa5]/
 export const isSimpleIdentifier = (name: string): boolean =>
   !nonIdentifierRE.test(name)
```

Comment:
I think we can refactor this part by using `propNameEscapeSymbolsRE` (move it to `shared` pkg)
https://github.com/vuejs/core/blob/caeb8a68811a1b0f799632582289fcf169fb673c/packages/compiler-sfc/src/script/utils.ts#L110

Reply:
seems that directly use this regx to instead will break other test units